src/components/ThreeDotsMenu/index.js

```diff
@@ -96,6 +96,7 @@ function ThreeDotsMenu({iconTooltip, icon, iconFill, iconStyles, onIconPress, me
                                 hidePopoverMenu();
                                 return;
                             }
+                            buttonRef.current.blur();
```

Comment:
This is needed to prevent react native modal to refocus threedots button on modal hide when user close modal by pressing esc:
<details><summary>Video:</summary>

https://github.com/Expensify/App/assets/114975543/0c82ef9b-d024-4b15-afe7-6324fdaca34c

</details>

This is because of trap focus of react native modal.

Reply:
agree with @tsa321

Reply:
@aimane-chnaif  Because I am using `onBackButtonPress={Modal.closeTop}` in BaseModal.tsx to fix an issue about: When user press three dots menu on attachment modal then press `esc` the attachment modal is closed. The current behavior in staging:

<details><summary>Pressing esc on three dots menu in attachment modal: </summary>

https://github.com/Expensify/App/assets/114975543/89ae77bc-c7d5-4cf6-9f56-8b702eaab4e8

</details>

After using `onBackButtonPress={Modal.closeTop}` : 
<details><summary>blue border appears: </summary>

https://github.com/Expensify/App/assets/114975543/31a45cfd-646f-4b67-b39b-8152ce2e9bd2

</details>